src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.lock.json

```diff
@@ -1878,33 +1878,7 @@
           "lib/dotnet/xunit.assert.dll": {}
         }
       },
-      "xunit.console.netcore/1.0.2-prerelease-00088": {
```

Comment:
Not sure why this reference got removed from the project.lock.json completely. Should be referencing version 100.

To test the build of BuildTools, I used a localpublish of xunit,console.netcore with the version set to 100 to the buildtools/packages folder, then rebuilt and localpublished everything to the CoreFX repo for testing.